Libraries/LibGUI/MultiView.cpp

```diff
@@ -117,22 +117,27 @@ MultiView::~MultiView()
 
 void MultiView::set_view_mode(ViewMode mode)
 {
+    // This method recursively calls itself through `action->set_checked`.
+    // The following check is not just for optimization; it prevents an infinite loop.
```

Comment:
FWIW this pattern is prevalent throughout the codebase, so it's not unique here :)